src/store/current-game/current-game.store.ts

```diff
@@ -168,6 +169,13 @@ export const CurrentGameStoreModule: Module<CurrentGameState, RootState> = {
         player.hasGrelottine = false;
       }
     },
+    addJarret(state, playerName: string): void {
```

Comment:
Not sure this mutation should be in this store

Reply:
I duplicated the behavior of the grelottine

Reply:
As we discussed, there is another task to extract the whole game engine, not only the rule runner.

This task should not be done in this PR.

Reply:
ok